sonoff/support.ino

```diff
@@ -875,6 +875,10 @@ void GetFeatures()
 #ifdef USE_TM1638
   feature_sns1 |= 0x80000000;  // xsns_28_tm1638.ino
 #endif
+#ifdef USE_VL53L1X
+  feature_sns1 |= 0x90000000;  // xsns_29_vl53l1x.ino
```

Comment:
@arendst is this just an arbitrary number?

Reply:
Should be a bit value. As all bits of feature_sns1 have been used continue with feature_sns2 using bit mask 0x00000001